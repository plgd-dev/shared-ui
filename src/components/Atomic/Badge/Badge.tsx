import RB_Badge from 'react-bootstrap/Badge'
import { Props } from './Badge.types'

const Badge = (props: Props) => {
    const { children, ...rest } = props
    return (
        <RB_Badge pill {...rest}>
            {children}
        </RB_Badge>
    )
}

Badge.displayName = 'Badge'

export default Badge
