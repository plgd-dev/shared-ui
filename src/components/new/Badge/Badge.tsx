import { FC } from 'react'
import RB_Badge from 'react-bootstrap/Badge'

const Badge: FC = (props) => {
    const { children, ...rest } = props
    return (
        <RB_Badge pill {...rest} bg='info'>
            {children}
        </RB_Badge>
    )
}

Badge.displayName = 'Badge'

export default Badge
