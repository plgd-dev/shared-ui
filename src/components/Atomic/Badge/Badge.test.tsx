import { render } from '@testing-library/react'
import Badge from './Badge'

describe('<Badge>', () => {
    it('render correctly - snapshot', () => {
        const { asFragment } = render(<Badge>Badge test</Badge>)

        expect(asFragment()).toMatchSnapshot()
    })
})
