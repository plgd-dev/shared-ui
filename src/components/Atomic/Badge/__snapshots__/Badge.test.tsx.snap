// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Badge> render correctly - snapshot 1`] = `
<DocumentFragment>
  <span
    class="badge rounded-pill bg-primary"
  >
    Badge test
  </span>
</DocumentFragment>
`;
